### PR TITLE
review notes of module 1 were fixed

### PR DESCRIPTION
**Pull request by aschugunov**

### CODE REVIEW

File: module-1/module-1.js [hunk 1]
```diff
@@ -106,66 +103,60 @@ const getArrayElementsPairs = function (array1, array2) {
...
-    expect(getArrayElementsPairs([1,2], ['a', 'b'])).to.eql([[1, "a"], [1, "b"], [2, "a"], [2, "b"]]);
+    expect(getArrayElementsPairs([1,2], ['a', 'b'])).to.eql([[1, 'a'], [1, 'b'], [2, 'a'], [2, 'b']]);
   });
 });
 
 // ==== Deep equal ====
 const deepEqual = function (obj1, obj2) {
   let isPrimitive = function (obj) {
-    return Object(obj) !== obj;
+    return typeof obj !== 'object';
```
Comment (andrewlistopadov) [inline module-1/module-1.js]: missed not null check

File: module-1/module-1.js [hunk 2]
```diff
@@ -106,66 +103,60 @@ const getArrayElementsPairs = function (array1, array2) {
...
 
 describe('deepEqual', () => {
-  let obj = {here: {is: "an"}, object: 2};
+  let obj = {here: {is: 'an'}, object: 2};
   it('should get two values and returns true only if they are the same value or are objects with the same properties, where the values of the properties are equal', () => {
-    expect(deepEqual(obj, obj)).to.eql(true);    
+    expect(deepEqual(obj, obj)).to.eql(true);
     expect(deepEqual(obj, {here: 1, object: 2})).to.eql(false);
-    expect(deepEqual(obj,  {here: {is: "an"}, object: 2})).to.eql(true);
+    expect(deepEqual(obj,  {here: {is: 'an'}, object: 2})).to.eql(true);
```
Comment (andrewlistopadov) [inline module-1/module-1.js]: to see why not null check is important(see above), please add the line

expect(deepEqual(obj, {here: {is: null}, object: 2})).to.eql(false);

Comment (aschugunov) [reply module-1/module-1.js]: Yes, obviously it is important. I just simplified solution.

Comment (aschugunov) [reply module-1/module-1.js]: okay, thanks, I added new tests and null check

**Pull request merged**